src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.product-card {
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: #060b35;
+
+    width: 100%;
+  }
+  &__code {
+    font-size: 10px;
+    color: #616070;
+    text-align: left;
+    margin-top: 4px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-top: 24px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    color: #616070;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: #060b35;
+    text-align: right;
+  }
+
+  &__button {
+    text-decoration: none;
+    width: 100%;
+    height: 40px;
+    margin-top: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #00acdc;
```

Comment:
The background color should be defined using a SCSS variable. Ensure that all main values are stored in a separate variables file as per the task requirements.